### ISSUE TITLE
Help in implementing

### ISSUE THREAD

**Issue by regiesetgo** [pavelk2/social-feed#103]
I tried all the steps, but it doesn't show any feeds. My webserver (Apache) is running.
I tried open it thru localhost, still no feeds. Am I missing something? Thanks for the response.

**Comment by mparker11** [comment]
Have you tried putting the template.html in the root of your website? I didn't realize I needed to and had the same issue.

**Comment by Jokinen** [comment]
You can also just edit the social-feed/js/jquery.socialfeed.js line 14 to

``` javascript
template: 'bower_components/social-feed/template.html', // a path to the template file
```

if you don't want to pull the file out of its component.

The above is if your project's file structure is in accordance with the readme file.

There'll be an error in your browser's console indicating that the file was not found, if it's a file path problem.

**Comment by digitalesherz** [comment]
That is my Error in my console:
Does anyone have help for me?

![bildschirmfoto 2015-07-07 um 16 48 57](https://cloud.githubusercontent.com/assets/10600007/8548933/23187ed6-24c8-11e5-84b2-5e0d52e648ef.png)

**Comment by pavelk2** [comment]
So from the errors you have it seems that the paths to the javascript components are not correct - could you please show two things:
- the file structure of your project
- the index.html file content

**Comment by regiesetgo** [comment]
![picture](https://cloud.githubusercontent.com/assets/13147198/8691406/c951a908-2af4-11e5-9380-be91acd6fdfd.png)
root of my website? where is that? I just downloaded the .zip file, extracted it to a folder in my desktop and changed the API IDs and secrets. I also tried extracting it to my "htdocs" folder in xampp, still no feeds.  Is there anything that I should do? sorry, I'm still learning the how to's.
